### ISSUE TITLE
Midterm Report Peer Review

### ISSUE THREAD

**Issue by dhr4ever** [ORIE4741/projects#75]
I really like your project Identifying Gender Based on Voice Samples! First of all, I think this is a pretty practical problem to be solved, which could be used and be profitable in many industries. If I were a project manager, I will probably invest on your project. In addition, I really like the Data Description part. It’s very clear, every new term is well defined. This part could help readers to have a better understanding about what kind of features you will be using. In our project, we didn’t explain every terms, so I think that is a little bit confusing for readers. We will add the data description part in our project in the future. And the third thing I like the most is, your future plan is pretty organized. It makes sense to try some L1 regularizer. I really look forward to seeing how your project will go!

Talking about the things needed to be improved, there are generally three factors. First, I wish to see more in details about the data analysis. Will it be possible to add more statistical data visualization? Secondly, your team might try some regression, and do more analysis about that. Finally, for the testing part, could you please add more details about how you got those results? In that way, readers can be convinced, right?

In conclusion, this is a good project! I pretty enjoyed reading it. Good job, Premdeep, Michael and Siva!
